navigation.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  createLocalizedPathnamesNavigation,
+  Pathnames
+} from 'next-intl/navigation';
+ 
+export const locales = ['en', 'ar'] as const;
+export const localePrefix = 'always'; // Default
+ 
+// The `pathnames` object holds pairs of internal
+// and external paths, separated by locale.
+export const pathnames = {
+  // If all locales use the same pathname, a
+  // single external path can be provided.
+  '/': '/',
+  '/login': '/login',
+  '/register': '/register',
+  '/yousef':'/yousef',
+  'forgot-password':'forgot-password',
```

Comment:
Why `/yousef`

Reply:
I use it as a dummy path to navigate it when the targeted path hasn't been created yet.

Reply:
Why not to add the targeted path with an empty page? I think it is better than adding new dummy routes, because the empty page will be filled later.